Sources/CircularProgress/protocol/MouseTrackable.swift

```diff
@@ -0,0 +1,25 @@
+#if os(macOS)
+import Cocoa
+/**
+Makes it possible to utilize mouseOver/mouseOut functionality on a view (Supports custom bezier path)
+*/
+protocol MouseTrackable:class{
```

Comment:
I don't really see the point in making this so loose when it can only be used with `NSView` anyway.

I would make it:

```suggestion
protocol MouseTrackable: NSView {
```

Reply:
- it's a protocol not a class, so it can't extend anything else than other protocols. NSView is not a protocol.
- `SomeProtocol:class` in swift just means, it doesn't apply to structs.